### ISSUE TITLE
Cannot scroll to the right in asset list

### ISSUE THREAD

**Issue by TineUser** [grokability/snipe-it#13884]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

The asset list is wider than the display. So you have to scroll to right. But there's no scrollbar at the bottom to scroll to the right. There's only a scrollbar to scroll down and up on the left side.

To scroll to the right you have to select text beginning on the visible part and then moving to the right.

I don't know if other lists have the same issue.

### Reproduction steps

1. Open asset list
2. At the bottom you don't see the scrollbar
3. You can only sroll to the right by selecting text and move to the right

Please see the screenshots regarding the points above.

### Expected behavior

There should be a scrollbar at the bottom.

### Screenshots

![image](https://github.com/snipe/snipe-it/assets/77736099/8f7e9e49-6032-4bca-a64c-91cff69762b1)
![image](https://github.com/snipe/snipe-it/assets/77736099/342e7860-a400-418b-9b36-66d98b8e8e4b) 

### Snipe-IT Version

6.2.3

### Operating System

Windows

### Web Server

Apache

### PHP Version

8.0.19

### Operating System

Windows

### Browser

Microsoft Edge (Chromium-based)

### Version

119.0.2151.58

### Error messages

```shell
No error message
```

### Additional context

Upgraded from fresh install with 6.2.2 to 6.2.3.
Windows 10 with XAMPP

**Comment by TineUser** [comment]
In some other lists there's a horizontal scrollbar, sometimes even 2 scrollbars:

![image](https://github.com/snipe/snipe-it/assets/77736099/60d70e3b-349a-4d73-8d17-257337b3f245)

**Comment by TineUser** [comment]
Now I found out that the horizontal scrollbar is at the bottom of the page. This isn't very usefull. You have first to scroll down to have a possibility to scroll horizontally. So you have then to scroll up again to come back to the asset but then you don't know where the asset is because you don't see the column with the asset name.

And you can't use the arrow keys to scroll horizontally. The arrow keys for scrolling vertically are working.

**Comment by snipe** [comment]
In most modern browsers, you can use a two-finger slide on your mousepad to scroll. 

There's just not a lot we can do about it AFAIK, since that's part of the bootstrap-tables functionality.

**Comment by TineUser** [comment]
Yes, that's right. The two-finger slide is working. I didn't know this option before.

I'm wondering why the horizontal scrollbar won't be shown at the bottom of the display, when the table is wider than the display resolution like it is the case with the vertical scrollbar?

**Comment by mrfusion71** [comment]
Good morning,

I have the same problem, but I can't use the two finger slide because most of the time I don't have a touchpad.
Is there any news about solving this problem?

Thank you very much

note: I use the release 8.0.3

**Comment by snipe** [comment]
There already is a scrollbar at the bottom: 

https://github.com/user-attachments/assets/80407d08-a522-4bbe-be98-1b40ce941273
